src/boot/efi/cpio.c

```diff
@@ -432,23 +434,40 @@ EFI_STATUS pack_cpio(
                 return log_error_status_stall(err, L"Failed to pack cpio trailer: %r");
 
         for (UINTN i = 0; i < n_tpm_pcr; i++) {
+                bool m;
+
+                if (tpm_pcr[i] == UINT32_MAX) /* Disabled */
```

Comment:
This is already done by tpm_log_event, why replicate?

Reply:
if we measure multiple things to a PCR and one measurement isn't done we want to treat that as incompletely measured, i.e. set ret_measured = false. But if we measure the same thing to multiple PCRs, and one of that isn't done because explicitly disabled, then we still want to consider the measurement as having taken place.

i.e. measurement to multiple PCRs is redundant, and it's fine if one of them is disabled.

measurement of multiple blocks where one is skipped is *not* redundant though, and we should handle that differently.